tests/ResponseBuilderTest.php

```diff
@@ -1,12 +1,22 @@
 <?php
 
-namespace Softonic\GraphQL\Test;
+namespace Softonic\GraphQL;
 
 use PHPUnit\Framework\TestCase;
-use Softonic\GraphQL\ResponseBuilder;
 
 class ResponseBuilderTest extends TestCase
 {
+    private $dataObjectBuilder;
+
+    private $responseBuilder;
+
+    protected function setUp()
+    {
+        $this->dataObjectBuilder = $this->createMock(DataObjectBuilder::class);
+
+        $this->responseBuilder = new ResponseBuilder($this->dataObjectBuilder);
+    }
+
     public function testBuildMalformedResponse()
```

Comment:
I'd use the same approach for identifying tests (using annotations)

Reply:
You're right, I didn't touch the existent ones.